openfl/component/straggler_handling_functions/straggler_handling_function.py

```diff
@@ -5,15 +5,49 @@
 
 from abc import ABC
 from abc import abstractmethod
+from typing import Callable
 
 
 class StragglerHandlingFunction(ABC):
```

Comment:
This is no longer a pure functional interface - I'd suggest to rename it to `StragglerHandlingPolicy`

Reply:
The class is renamed as suggested. DISCUSS: In our opinion name of the file and sub-package should be renamed. If this is acceptable then this can be done in a follow-up PR.

Reply:
Makes sense - as discussed on our sync meeting, it's ok to continue the clean-up in a separate PR.